draft-ietf-quic-load-balancers.md

```diff
@@ -1011,9 +935,8 @@ same server. It could then apply analytical techniques to try to obtain the
 server encoding.
 
 The Stream and Block Cipher CID algorithms provide robust entropy to making any
-sort of linkage.  The Obfuscated CID obscures the mapping and prevents trivial
-brute-force attacks to determine the routing parameters, but does not provide
-robust protection against sophisticated attacks.
+sort of linkage. The Plaintext CID algorithm makes no attempt to protect this
+encoding.
 
```

Comment:
Are you sure that "provide robust entropy to making any sort of linkage" is a valid English sentence? Would it be better to say "provide robust entropy to defend against any sort of linkage", or something like that?